src/main/java/me/jellysquid/mods/sodium/client/util/workarounds/probe/GraphicsAdapterVendor.java

```diff
@@ -10,11 +10,11 @@ public enum GraphicsAdapterVendor {
 
     @NotNull
     static GraphicsAdapterVendor identifyVendorFromString(String vendor) {
-        if (vendor.startsWith("Advanced Micro Devices, Inc.") || vendor.contains("(0x1002)")) {
+        if (vendor.startsWith("Advanced Micro Devices, Inc.") || vendor.contains("(0x1002)") || vendor.equals("0x1002")) {
             return AMD;
-        } else if (vendor.startsWith("NVIDIA") || vendor.contains("(0x10de)")) {
+        } else if (vendor.startsWith("NVIDIA") || vendor.contains("(0x10de)") || vendor.equals("0x10de")) {
             return NVIDIA;
-        } else if (vendor.startsWith("Intel") || vendor.contains("(0x8086)")) {
+        } else if (vendor.startsWith("Intel") || vendor.contains("(0x8086)") || vendor.equals("0x8086")) {
```

Comment:
Is there a reason this can't be replaced with `vendor.contains("0xDRVR")`? e.g. `if (vendor.startsWith("Advanced Micro Devices, Inc.") || vendor.contains("0x1002"))`

Reply:
I originally didn't replace it to be on the safer side, but I suppose it's a safe assumption that the vendor ID won't be part of some longer unrelated string